# Own's Libraries
from errors.base_error import BaseError


class UtilError(BaseError):
    pass
